src/egon/data/datasets/power_plants/mastr_db_classes.py

```diff
@@ -280,3 +295,154 @@ class EgonPowerPlantsStorage(Base):
+        "wind": EgonPowerPlantsWind,
+        "biomass": EgonPowerPlantsBiomass,
+        "hydro": EgonPowerPlantsHydro,
+        "combustion": EgonPowerPlantsCombustion,
+        "gsgk": EgonPowerPlantsGsgk,
+        "nuclear": EgonPowerPlantsNuclear,
+        "storage": EgonPowerPlantsStorage,
+    }
+
+    deposit_id_data_bundle = config.datasets()["data-bundle"]["sources"][
+        "zenodo"
+    ]["deposit_id"]
+    deposit_id_mastr = config.datasets()["mastr_new"]["deposit_id"]
+
+    contris = contributors(["kh", "kh"])
+
+    contris[0]["date"] = "2023-03-15"
+
+    contris[0]["object"] = "metadata"
+    contris[1]["object"] = "dataset"
+
+    contris[0]["comment"] = "Add metadata to dataset."
+    contris[1]["comment"] = "Add workflow to generate dataset."
+
+    for technology in technologies:
+        target_table = target_tables[technology]
+
+        meta = {
+            "name": (
+                f"{target_table.__table_args__['schema']}."
+                f"{target_table.__tablename__}"
+            ),
+            "title": f"eGon {technology} power plants",
+            "id": "WILL_BE_SET_AT_PUBLICATION",
+            "description": (
+                f"eGon {technology} power plants status quo derived from MaStR"
+            ),
+            "language": "en-US",
+            "keywords": [technology, "mastr"],
+            "publicationDate": datetime.date.today().isoformat(),
+            "context": context(),
+            "spatial": {
+                "location": "none",
+                "extent": "Germany",
+                "resolution": "1 m",
+            },
+            "temporal": {
+                "referenceDate": (
+                    config.datasets()["mastr_new"]["egon2021_date_max"].split(
+                        " "
+                    )[0]
+                ),
+                "timeseries": {},
+            },
+            "sources": [
+                {
+                    "title": "Data bundle for egon-data",
+                    "description": (
+                        "Data bundle for egon-data: A transparent and "
+                        "reproducible data processing pipeline for energy "
+                        "system modeling"
+                    ),
+                    "path": (
+                        "https://sandbox.zenodo.org/record/"
+                        f"{deposit_id_data_bundle}#.Y_dWM4CZMVM"
+                    ),
+                    "licenses": [license_dedl(attribution="© Cußmann, Ilka")],
+                },
+                {
+                    "title": (
+                        "open-MaStR power unit registry for eGo^n project"
+                    ),
+                    "description": (
+                        "Data from Marktstammdatenregister (MaStR) data using "
+                        "the data dump from 2022-11-17 for eGon-data."
+                    ),
+                    "path": (
+                        f"https://sandbox.zenodo.org/record/{deposit_id_mastr}"
```

Comment:
The data on the zenodo sandbox was removed several months ago. It is now uploaded in the "normal" zenodo. 
That change is already on dev, but when we created the tables that we will publish, it was still on the sandbox. 
I am not sure if we should do anything about it. It is also not only related to the metadata you added in this PR, since the data bundle is used often. 
What do you think?

Reply:
It is the same question for the data bundle and the mastr data on zenodo.

Reply:
> The data on the zenodo sandbox was removed several months ago. It is now uploaded in the "normal" zenodo. That change is already on dev, but when we created the tables that we will publish, it was still on the sandbox. I am not sure if we should do anything about it. It is also not only related to the metadata you added in this PR, since the data bundle is used often. What do you think?

For now I'll fix it only in the metadata strings and not the functions to avoid merge conflicts. We'll need to merge the dev into #450 and test it's functionality in the end.